src/overview/using-the-standard-library.md

```diff
@@ -1,63 +1,34 @@
-# Using the Rust Standard Library (`std`)
+# Using the Standard Library (`std`)
 
-Espressif provides a C-based development framework called [esp-idf] which has support for all Espressif chips starting with the ESP32; note that this framework does _not_ support the ESP8266.
+Espressif provides a C-based development framework called [esp-idf][esp-idf-github] which has support for all Espressif chips starting with the ESP32; note that this framework does _not_ support the ESP8266. See also Section [Support for Espressif Products][rust-esp-book-std-vs-no-std-support].
 
-`esp-idf` in turn provides a [newlib] environment with enough functionality to build the Rust standard library (`std`) on top of it. This is the approach that is being taken to enable `std` support on ESP devices.
+[rust-esp-book-std-vs-no-std-support]: ./comparing-std-and-no_std.html#support-for-espressif-products
 
-## Chip Support
+`esp-idf` in turn provides a [newlib][newlib-env] environment with enough functionality to build the Rust standard library (`std`) on top of it. This is the approach that is being taken to enable `std` support on ESP devices.
```

Comment:
We need to decide how to format esp-idf, as some times is `esp-idf` and others esp-idf

Reply:
Sometimes it is also ESP-IDF.

We will address this issue in our style guide and then will make respective corrections.